contrib/containerized-pbench/README.md

```diff
@@ -1,81 +1,118 @@
-# containerized-pbench
-
-## Openshift
-
-We can either build an image using the following commands or pull the image from dockerhub
-
-Build the image from source like
+## Containerized pbench
+This directory contains OPenShift templates, dockerfiles needed to build and run containerized pbench.
```

Comment:
OpenShift (actually, I'm not sure how to capitalize it, but I'm pretty sure the P should be lower case :-) )